stroom-core-server/src/main/java/stroom/connectors/kafka/StroomKafkaProducerFactoryService.java

```diff
@@ -32,14 +35,14 @@
     private static final String DEFAULT_NAME = "default";
 
     // Old configs may not have named server/version properties, so use this property name
-    private static final String PROP_BOOTSTRAP_SERVERS = "stroom.kafka.bootstrap.servers";
+    private static final String OLD_PROP_BOOTSTRAP_SERVERS = "stroom.kafka.bootstrap.servers";
 
     // Default kafka client version for old configs
     private static final String KAFKA_CLIENT_VERSION_DEFAULT = "0.10.0.1";
 
```

Comment:
The defaults for all our properties live in stroomCorePropertyContext.  Have you set them in there?  Also there is an  entry for the bootstrap servers in stroom.conf.example in the root of the repo.